requirements.txt

```diff
@@ -1,10 +1,9 @@
 PyStemmer>=1.3,<2.0
-tensorflow>=1.5,<2.0
 bblfsh>=2.2.1,<3.0
 modelforge>=0.5.4-alpha,<0.6
-sourced-engine>=0.5.1,<0.6
+sourced-engine>=0.5.1,<0.7
```

Comment:
Let's require a specific version as it is not very stable lately

Reply:
0.6.1 has been working for me, I guess we might as well use that one (it also fixes some bug when Siva files are corrupt)